.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
In other projects we add this to `eslintConfig` in `package.json`, so could be worth doing here.

Reply:
I'm quite sure the current config doesn't use @evrythng/eslint-config-evrythng, if so, there will be a lot of error. For now let use the standard config. We will update it later

Reply:
This is fine. Standard was the choice two+ years ago and we can move to our Airbnb style one at a later date.

Reply:
I moved eslint config file to package.json but did not install our @evrythng/eslint-config-evrythng dependency and used standard because as you said there were too much errors after running lint with @evrythng/eslint-config-evrythng

Reply:
Also it's a private package, which the public could not install if they use this OSS repo.